conda-env.yml

```diff
@@ -0,0 +1,7 @@
+dependencies:
```

Comment:
Do we need this file? Isn't it better to run `pip install -r requirements.txt` inside the environment

Reply:
PyPi and Conda are two different ecosystems. Sometimes the packages have different names (brotli vs. brotli-python). And while PyPi has wheels, Conda has it's own packages for supporting libraries. Using Pip with Conda is not really recommended. That's why I came up with a separate `build_conda`. And this file seems required in the setup of `conda-incubator/setup-miniconda@v2`.

Reply:
Requirement files are partially understood by `conda`:
```
$ conda install --file requirements.txt
CondaValueError: could not parse 'brotli; platform_python_implementation=='CPython'' in: requirements.txt
```
But that's actually two errors. If the amendment were understood, it would install `brotli` (the command-line programs) and not `brotli-python` (the python bindings).

It should be possible to hardcode the package list with
```
conda install mutagen pycryptodomex websockets brotli-python certifi pyinstaller
```
and then remove this file and the `environment-file: conda-env.yml` line.

Reply:
Ok, hardcoding the package list works. If you think that should be done, tell me and I will update this PR.

Merging `build_unix` and `build_conda` does not seem sensible to me and I have no idea how it should be done.

Reply:
I wouldn't have insisted on generalizing if it's complicated. But I like what you have done rn